src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -344,27 +344,20 @@ public object GetParameterValueByName(string parameterName)
         }
 
         /// <summary>
-        /// Override the element's color in the active view.
+        ///     Override Elements Graphics Settings in Active View.
         /// </summary>
-        /// <param name="color">The color to apply to a solid fill on the element.</param>
-        public Element OverrideColorInView(Color color)
```

Comment:
please don't remove this public function. For now, I would like to keep both versions, removing a public node method will break old graphs, and I'd rather not figure out how to migrate this node in this PR.